layers/+lang/latex/packages.el

```diff
@@ -77,10 +77,16 @@
         "mv" 'TeX-view))))
 
 (when (string= latex-build-command "LatexMk")
-(defun latex/init-auctex-latexmk ()
-  (use-package auctex-latexmk
-    :defer t
-    :init (add-hook 'LaTeX-mode-hook 'auctex-latexmk-setup))))
+  (defun latex/init-auctex-latexmk ()
+    (use-package auctex-latexmk
+      :defer t
+      :init
+      (progn
+        (add-hook 'LaTeX-mode-hook
+                  (lambda()
+                    (setq TeX-command-default "LatexMk")))
```

Comment:
I tested it out and just setting it once doesn't work. Not sure where exactly it gets overwritten, but it does. Adding it to `LaTeX-mode-hook` is the only way I can see to make it actually work. Of course I'm happy if someone can see a cleaner way.